.pr_agent.toml

```diff
@@ -0,0 +1,39 @@
+[config]
+model="gpt-4" # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+model_turbo="gpt-4" # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+fallback_models=["gpt-4"] # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
```

Comment:
How did we pick the models, is it just whatever's latest? Does `fallback_models` need to include something that's different from the first choices?

Reply:
I chose because from the text type models. There are some that are image/audio specific. I think we can choose multiple but I *think* we would need to host those models too or play around with it and see how it works.